### ISSUE TITLE
bug

### ISSUE THREAD

**Issue by xiong-er** [CHLNDDEV/OceanMesh2D#107]
hello,my friend:
          I downloaded the compression package and unzipped it under my MATLAB/Toolbox  folder. I opened MATLAB and added the path. The software showed an error: the folder starting with @ could not be added to the path.
So I changed all the folder names with @ (such as @ann to Ann), but could not add them, showing that MATLAB could not save the changes to the modified path,pathdef.m may be a read-only file.I added each folder separately and found that it was the Ann folder that made the error.
        I tried many methods but could not solve it. I installed other MATLAB tools normally, and the MATLAB I used was 2017A version. Hope to get your answer or email, thank you very much.

**Comment by krober10nd** [comment]
This isn't a bug. You installed the software incorrectly. Do not place it in the MATLAB root path. Unzip the folder on your desktop or in some other place easily found. 

Please do not change the names of the folders otherwise things will not work.

**Comment by xiong-er** [comment]
Hello,friend.I am&nbsp; glad&nbsp;to&nbsp;receive&nbsp;your&nbsp;reply.
I've&nbsp;tried&nbsp;unzipping&nbsp;files&nbsp;on&nbsp;the&nbsp;desktop,&nbsp;but&nbsp;the&nbsp;problem&nbsp;is&nbsp;the&nbsp;same.I&nbsp;first&nbsp;tried&nbsp;to&nbsp;add&nbsp;a&nbsp;path&nbsp;into&nbsp;my&nbsp;MATLAB&nbsp;without&nbsp;changing&nbsp;the&nbsp;file&nbsp;name,&nbsp;but&nbsp;the&nbsp;MATLAB&nbsp;was&nbsp;read&nbsp;as&nbsp;follows&nbsp;(no&nbsp;@ann,&nbsp;@msh...File&nbsp;path) :picture1.

&nbsp;&nbsp;






Then&nbsp;I&nbsp;can&nbsp;only&nbsp;try&nbsp;to&nbsp;change&nbsp;the&nbsp;file&nbsp;name&nbsp;and&nbsp;remove&nbsp;the&nbsp;@&nbsp;symbol,&nbsp;but&nbsp;the&nbsp;folder&nbsp;of&nbsp;Ann&nbsp;still&nbsp;shows&nbsp;the&nbsp;following&nbsp;error（picture2）:


I&nbsp;don't&nbsp;know&nbsp;if&nbsp;there&nbsp;is&nbsp;something&nbsp;wrong&nbsp;with&nbsp;my&nbsp;compression&nbsp;package.&nbsp;I&nbsp;downloaded&nbsp;the&nbsp;file&nbsp;three&nbsp;times&nbsp;before&nbsp;and&nbsp;unzipped&nbsp;it&nbsp;and&nbsp;said&nbsp;there&nbsp;was&nbsp;an&nbsp;error.&nbsp;The&nbsp;fourth&nbsp;time&nbsp;I&nbsp;downloaded&nbsp;the&nbsp;file&nbsp;successfully&nbsp;without&nbsp;any&nbsp;error.
But&nbsp;I&nbsp;compared&nbsp;the&nbsp;following&nbsp;on&nbsp;GitHub&nbsp;with&nbsp;no&nbsp;problems,&nbsp;it&nbsp;was&nbsp;no&nbsp;problem&nbsp;installing&nbsp;other&nbsp;MATLAB&nbsp;kits,&nbsp;and&nbsp;it&nbsp;was&nbsp;no&nbsp;problem&nbsp;trying&nbsp;to&nbsp;use&nbsp;someone&nbsp;else's&nbsp;computer.
I have tried your method,and&nbsp;&nbsp;I&nbsp;don't&nbsp;know&nbsp;where&nbsp;the&nbsp;problem&nbsp;is&nbsp;right&nbsp;now.

------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "CHLNDDEV/OceanMesh2D"                                                                                    <notifications@github.com&gt;;
发送时间:&nbsp;2020年8月22日(星期六) 中午11:20
收件人:&nbsp;"CHLNDDEV/OceanMesh2D"<OceanMesh2D@noreply.github.com&gt;;
抄送:&nbsp;"中国科学院大学 熊杰"<1759550416@qq.com&gt;;"Author"<author@noreply.github.com&gt;;
主题:&nbsp;Re: [CHLNDDEV/OceanMesh2D] bug (#107)





 
This isn't a bug. You installed the software incorrectly. Do not place it in the MATLAB root path. Unzip the folder on your desktop or in some other place easily found.
 
Please do not change the names of the folders otherwise things will not work.

**Comment by krober10nd** [comment]
I can't understand any of what is written above.

**Comment by xiong-er** [comment]
Hello,friend.I am  glad to receive your reply and Sorry for the mess above.
I've tried unzipping files on the desktop, but the problem is the same.I first tried to add a path into my MATLAB without changing the file name, but MATLAB cannot add the @ symbol folder into the path (no @ann, @msh...File path) 
  

Then I can only try to change the file name and remove the @ symbol, but the folder of Ann still shows the following error:

I don't know if there is something wrong with my compression package. I downloaded the file three times before and unzipped it and said there was an error. The fourth time I downloaded the file successfully without any error.
But I compared the following on GitHub with no problems, it was no problem installing other MATLAB kits, and it was no problem trying to use someone else's computer.
I have tried your method,and  I don't know where the problem is right now.
